### DISCUSSION TITLE
dotnet watch in custom module project

### DISCUSSION THREAD

**Discussion by PizzaConsole** [oqtane/oqtane.framework#710] [General]
I am very interested to know if there is a way to get dotnet watch working while I am working in custom module project so I do not have to constantly rebuild and run to see my changes.

**Comment by sbwalker** [comment]
The only thing I have seen related to this is: https://medium.com/@sefatanam/blazor-enable-auto-reload-or-hot-loading-in-client-side-and-server-side-asp-net-5f6ca38d6d80 - but I have not tried to myself. There are supposedly some improvements for dotnet watch coming in .NET 5 but I have not had time to explore those yet either.
